packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -206,6 +206,7 @@ class FlutterDriverExtension {
 
   /// Runs `finder` repeatedly until it finds one or more [Element]s.
   Future<Finder> _waitForElement(Finder finder) async {
+    await null; // TODO(mravn): This method depends on async execution.
```

Comment:
Looks like you need to add an `// ignore ...` here for the "await only futures" lint